index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Document</title>
+  </head>
```

Comment:
- [ ] Great job so far. like the previous reviewer noted, kindly make your design look like the figma version. your hamburger icon is wrongly placed
your version
![Screenshot (211)_LI](https://user-images.githubusercontent.com/65298048/177529502-0f2669b2-19b8-4af6-a6a8-9d71e25d8477.jpg)
figma version
![Screenshot (212)_LI](https://user-images.githubusercontent.com/65298048/177529533-1e3c561b-92c7-448e-b999-a503379ca416.jpg)